src/views/Proposals.vue

```diff
@@ -103,7 +103,8 @@ export default {
 
             if (
               ['core', 'all'].includes(this.selectedState) &&
-              this.space.core.includes(proposal[1].address)
+              this.space.core.includes(proposal[1].address) &&
+              !this.space.invalid.includes(proposal[1].authorIpfsHash)
```

Comment:
It's better if you move the lines here https://github.com/balancer-labs/snapshot/pull/124/files#diff-c970d6004452ce92680f4993b4befde0R111-R115 inside the first condition here: https://github.com/balancer-labs/snapshot/pull/124/files#diff-c970d6004452ce92680f4993b4befde0R98-R102 this way you will already filterout the invalid without having to add more conditions later